### PR TITLE
Double-check Event schema

### PR DESCRIPTION
**Pull request by paultag**
- Include a few missing fields
- Add `timezone`
- Update JSON Schema

### CODE REVIEW

**Comment by paultag** [comment]
Sorry for the noise, this also removes the descriptions (since that's what the OCDEP is for)

File: proposals/0004.rst [hunk 1]
```diff
@@ -37,17 +37,38 @@ Core Fields
...
+    Description of the Event.
+
+location
+    Location object, as defined in the Location section below.
+
 all_day
     Boolean value set to boolean ``True`` if the event is an all-day event, otherwise it must be set to the boolean value ``False``.
 
 end_time
-    Fully timezone aware datetime set to the end time of the event.
```
Comment (jamesturk) [inline proposals/0004.rst:50]: this seems ambiguous, why would it be in either form?  we should define the exact form (ISO 8601?)

Comment (paultag) [inline proposals/0004.rst:50]: Hurm, yeah. Good point. OK.

I wonder what this means for the `timezone` field. If we're always encoding it in ISO8601, that has Timezone support. I guess it is a good way to ensure we always know local time, and always validate it has TZ everywhere? Thoughts?

cc/ @twneale @jamesturk

**Comment by jamesturk** [comment]
could we just normalize everything to UTC & append 'Z' to the 8601 formatted date?

**Comment by paultag** [comment]
Absolutely. Everyone cool with that definition?

**Comment by jamesturk** [comment]
mind updating the PR so I can read over it all together?

**Comment by paultag** [comment]
@jamesturk updated

**Pull request merged**